validate.py

```diff
@@ -85,6 +86,34 @@ def get_reviews():
 def get_users():
   return list(sorted(os.listdir('players/')))
 
+def get_user_points():
+  points = {}
+
+  for user in get_users():
+    points[user] = 0
+
+    with open(os.path.join('players', user)) as inf:
+      try:
+        points[user] += int(inf.read())
+      except ValueError:
```

Comment:
```suggestion
      except:
```

In general, what are the rules around an uncaught non-player foo wins Exception?

Reply:
Specifically, here I'm not 100% sure whether ValueError is the only exception type that can be triggered by int() or inf.read() and can't tell if this is a way that Jeff is trying to sneakily win by causing an exotic exception.